source/deployment/on-boarding.rst

```diff
@@ -67,3 +67,77 @@ Common Tasks
  - Follow the steps outlined in the `upgrade documentation <https://docs.mattermost.com/administration/upgrade.html>`_ to perform upgrades.   
+
+User Experience Optimizations
+-----------------------------
+
+We highly recommend the following best practices, configuration options, and features for an optimal Mattermost user experience.
+
+**1. Upgrade your Mattermost Server**
+
+Frequent server upgrades are critical to maintain an optimized user experience. When you upgrade your Mattermost server frequently, your users can access new features, improved user experiences, bug fixes, and guaranteed mobile app compatibility. Upgrading your Mattermost server only takes a few minutes. See the `Upgrade Guide <https://docs.mattermost.com/administration/upgrade.html>`__ for step-by-step instructions. 
+
+**2. Install Plugins**
+
+Plugins customize and extend the Mattermost platform. Install plugins with a single click from the **Main Menu > Plugin Marketplace**. See the `Plugin Marketplace  <https://docs.mattermost.com/administration/plugins.html#plugin-marketplace>`__ documentation for details.
+
+  Consider installing and enabling the following productivity plugins for your users:   
+    - Enable audio/video calling and screen sharing with `Jitsi <https://integrations.mattermost.com/?s=Jitsi%20Plugin>`__ or `Zoom <https://integrations.mattermost.com/?s=Zoom%20Plugin>`__.
+    - Create polls with `Matterpoll <https://integrations.mattermost.com/?s=Matterpoll%20Plugin>`__.
+    - Share GIFs with `GIF Commands <https://integrations.mattermost.com/?s=gif&submit=>`__.
+    - Create and share memes with `Memes <https://integrations.mattermost.com/?s=Memes%20Plugin>`__.
+    - Set personal reminders with `Remind <https://integrations.mattermost.com/?s=Remind%20Plugin>`__.
+    - Create and share to do items with `Todo <https://github.com/mattermost/mattermost-plugin-todo>`__.
+    - Customize welcome messages for new users with `WelcomeBot <https://integrations.mattermost.com/?s=WelcomeBot%20Plugin>`__.
+
+Explore all plugins and integrations available in the `Mattermost Plugin Marketplace <https://integrations.mattermost.com/>`__.
+
+**3. Enable Automatically Extended Sessions**
+
+Keep your desktop and mobile users logged in and refresh user sessions automatically by configuring the following: **System Console > Sessions > Extend session length with activity**: ``true``. See the `Extend session length with activity <https://docs.mattermost.com/administration/config-settings.html#extend-session-length-with-activity>`__ configuration settings documentation for details. 
```

Comment:
Propose adding a link to blog for more details

Reply:
Sounds good, thanks!